pallets/staking/src/lib.rs

```diff
@@ -1288,13 +1244,12 @@ decl_storage! {
                     T::Currency::free_balance(&stash) >= balance,
                     "Stash does not have enough balance to bond."
                 );
-                let controller_origin = <Module<T>>::get_origin(controller.clone());
-                <Module<T>>::bond(
-                    <Module<T>>::get_origin(stash.clone()),
+                let _ = <Module<T>>::bond(
+					T::Origin::from(Some(stash.clone()).into()),
                     T::Lookup::unlookup(controller.clone()),
                     balance,
                     RewardDestination::Staked,
-                ).expect("Unable to bond");
```

Comment:
Well I guess substrate's idea is that we don't want to panic if bonding cannot happen. I figure it doesn't matter anymore, since genesis block has already happened, so we might as well align with substrate.